### PR TITLE
Homepage

### LINKED ISSUE #51: Homepage
As a pet owner, I want to be greeted with a homepage where I can connect to the car when I log into the app so that I can quickly start playing.

Acceptance criteria
- The user can connect to the car on the homepage 
- The user is welcomed when they log into the app

### PR DESCRIPTION
**Pull request by AstridBer**
Moved the controlpanel to the playpage and created a homepage with a welcome text and a button to connect to MQTT 

resolves #51

### CODE REVIEW

File: smart_pet_buddy/lib/homepage.dart [hunk 1]
```diff
@@ -17,39 +19,101 @@ class HomePage extends StatefulWidget {
...
+                  crossAxisAlignment: CrossAxisAlignment.center,
+                  children: [
+                    Text(
+                      'Welcome!',
+                      style: TextStyle(
+                        fontSize: 40,
+                      ),
+                    ),
+                    Text(
+                      'To start playing please connect to the car :)',
```
Comment (emilieanthony) [inline smart_pet_buddy/lib/homepage.dart:79]: Nice that you added this text! I think this is very helpful for the user and improves the user experience a lot.

File: smart_pet_buddy/pubspec.lock [hunk 2]
```diff
@@ -406,6 +406,27 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "2.1.0"
+  video_player:
```
Comment (emilieanthony) [inline smart_pet_buddy/pubspec.lock:409]: Nice, does this allow us to have a video on the homepage?

Comment (AstridBer) [reply smart_pet_buddy/pubspec.lock:409]: Nope, I tried to but gave up and just added a picture instead😅 I'll remove that part

**Review by emilieanthony — APPROVED** [review]
Nice work on the homepage!

**Pull request merged**